kura/distrib/src/main/resources/generic-aarch64/deb/control/control

```diff
@@ -19,7 +19,7 @@ Depends: openjdk-8-jre-headless | temurin-8-jdk | openjdk-17-jre-headless | temu
  polkit | policykit-1, ssh | openssh, openssl, busybox, openvpn,
  bluez | bluez5, bluez-hcidump | bluez5-noinst-tools,
  ntpdate, chrony, chronyc | chrony, cron | cronie,
- network-manager | networkmanager, bind9 | bind, isc-dhcp-server | dhcp-server, isc-dhcp-server | dhcp-client, iw, iptables, modemmanager,
+ network-manager | networkmanager, bind9 | bind, dnsmasq | isc-dhcp-server | dhcp-server, dnsmasq | isc-dhcp-server | dhcp-client, iw, iptables, modemmanager,
```

Comment:
if we add the dnsmasq to the generic profiles, does it make sense to keep the old isc-dhcp-server and dhcp-server code?

Reply:
Not really, but that way we are excluding devices that have isc-dhcp-server but not dnsmasq. I don't know if any of the supported devices fall in such case.